src/Libraries/RevitNodesUI/RevitNodesUI.csproj

```diff
@@ -36,7 +36,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="DynamoUtilities">
-      <HintPath>$(DYNAMOAPI)\DynamoUtilities.dll</HintPath>
+      <HintPath>..\..\..\..\..\..\..\..\Program Files\Dynamo\Dynamo Core\1.2\DynamoUtilities.dll</HintPath>
```

Comment:
Once $(DYNAMOAPI) variable is defined correct then all the references will resolve automatically. You need to close the solution and reopen after making changes to csprops file.